server/controller/index.js

```diff
@@ -57,3 +57,32 @@ exports.getId = async (ctx) => {
       return ctx.body;
     });
 };
+
+
+exports.markSpam = async (ctx) => {
+  ctx.user = { id: '5a2ff257fc13ae6d59000535' };
```

Comment:
We will be using `koa-passport` package later. So I believe user information will be saved as `ctx.state.user` could you update the function to use a single const with information saved from `ctx.state.user.id`.